webapp/locales/de.json

```diff
@@ -354,6 +354,7 @@
   },
   "post": {
     "name": "Beitrag",
+    "pinned": "Gepinnt",
```

Comment:
Here I would prefer a more funktional description from the users perspective like `System info` or similar …
If the user reads `Pinned` or `Gepinnt` then they may think: Oh I haven’t pinned a post …
Or?